version1/02 - Object types.md

```diff
@@ -90,863 +92,614 @@ The following notation is used in the tables below:
-|           | ref       | refKryssreferanseFraMappe        |          |             |                          |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktMappe** that cross reference the current **AbstraktRegistrering**                                                                                                                                                                                                                                       |
-|           | ref       | refKryssreferanseTilRegistrering |          |             |                          |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktRegistrering** that the current **AbstraktRegistrering** cross references                                                                                                                                                                                                                               |
-|           | ref       | refKryssreferanseFraRegistrering |          |             |                          |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktRegistrering** that cross reference the current **AbstraktRegistrering**                                                                                                                                                                                                                                |
-
-### Arkivnotat
+### Basisregistrering ([Finalizable Object](#finalizable-object-basic-archival-object))
+(Basic Record)
+
+**Basisregistrering** introduces the common fields for all *registrering* objects.
+
+Note: **EFO** required if finalized
+
+| Field no. | Type      | Field                            | Not null |    Default    | Code list          |  | Save | View | Query | Sort | Link | Unlink | Comment                                                                                                                                                                                                                                                                  |
+|:----------|:----------|:---------------------------------|:--------:|:-------------:|:-------------------|:-|:----:|:----:|:-----:|:----:|:----:|:------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| M004      | string    | registreringsIdent               |    V     | Auto inc      |                    |  |  cw  |  V   |   V   |  V   |      |        |                                                                                                                                                                                                                                                                          |
```

Comment:
As with the file type, I would not say that it is auto-incremented. I'd say that it is the `id` for some types and record year/record sequence number for others. We can describe a general default here and state that it may be overridden in children (as is the case with registry entry)